README.md

```diff
@@ -205,6 +205,8 @@ errors. The [@sentry/rollup-plugin](#) is used to upload source maps.
 - Preview links for non-default organizations work only after switching to the non-default
   organization in the Confluent Cloud UI in your browser.
 - When using multiple users on a single machine, only one user can run the extension at a time.
+- Searching by schema ID (in the Topics and Schemas views) is not supported due to cost and
```

Comment:
Perhaps instead, describe what facets are searchable? Else I might think that other aspects of either schemas or topics may be searchable, but aren't.

Reply:
Makes sense - updated in https://github.com/confluentinc/vscode/pull/1012/commits/a17985999679c5b14c49d0c3f51ad9e7185723da to lead with `subject` being the only searchable property